xdsl/dialects/arith.py

```diff
@@ -619,6 +619,42 @@ def get(op: SSAValue | Operation, target_type: AnyFloat):
         return TruncFOp.build(operands=[op], result_types=[target_type])
 
 
+@irdl_op_definition
+class TruncIOp(IRDLOperation):
+    name = "arith.trunci"
+
+    input: Operand = operand_def(IntegerType)
+    result: OpResult = result_def(IntegerType)
+
+    @staticmethod
+    def get(op: SSAValue | Operation, target_type: IntegerType):
```

Comment:
Can you please use inits? We haven't migrated every file yet but I feel that new operations should use initialisers

Reply:
Oh yes! The only change would be to have `def __init__(self, ...)` instead of `get` and the `staticmethod`!